schema/apply/action.yml

```diff
@@ -30,6 +30,10 @@ inputs:
     description: |
       URL(s) of the desired schema state.
     required: false
+  url:
+    description: |
+      The URL of the target database to apply changes to. For example: `mysql://root:pass@localhost:3306/prod`.
+    required: false
```

Comment:
Just missing here, the code already handled:

https://github.com/ariga/atlas-action/blob/master/atlasaction/action.go#L663